docs/proposals/control-plane-config.md

```diff
@@ -33,20 +36,24 @@ or `edit` the configuration.
 Proposed configuration CRD example:
 
 ```yaml
-apiVersion: nginx.gateway.k8s.io/v1beta1
+apiVersion: nginx.gateway.k8s.io/v1alpha1
```

Comment:
why `k8s.io` domain, not ours like `nginx.org` ?

Reply:
That was just based off of the gateway API group, but I'll be changing this to `gateway.nginx.org`.